INTMDB-655: PAK Resource Updates + Doc Cleanup + Deprecation Warnings (#1208)
mongodb/terraform-provider-mongodbatlas: website/docs/d/access_list_api_key.html.markdown

page_title: "MongoDB Atlas: access_list_api_key"
sidebar_current: "docs-mongodbatlas-datasource-access-list-api-key"
description: |-

>     Displays the access list entries for the specified Atlas Organization API Key. Atlas resources require that all API requests originate from IP addresses on the API access list.

Reviewer: See comment for resource about access lists and the Admin API.

Reviewer: ```suggestion
    Displays the access list entries for the specified Atlas Organization API Key. 
```
Making same suggestion as above.  Not blocking but nice to correct.

## Argument Reference

* `org_id` - (Required) Unique 24-hexadecimal digit string that identifies the organization that contains your projects.
* `cidr_block` - (Optional) Range of IP addresses in CIDR notation to be added to the access list.
* `ip_address` - (Optional) Single IP address to be added to the access list.
* `api_key_id` - (Required) Unique identifier for the Organization API Key for which you want to retrieve an access list entry.
* 

> -> **NOTE:** One of the following attributes must set: `cidr_block`  or `ip_address` not both.

Reviewer 2: I think adding a comma to "...cidr_block  or ip_address`,` not both" makes this a little clearer, but I suggest rephrasing to get rid of the colon:
```suggestion
-> **NOTE:** You must set either the `cidr_block` attribute or the `ip_address` attribute. Don't set both.
```
